engine/src/main/java/org/hibernate/validator/internal/util/logging/formatter/CollectionOfClassesObjectFormatter.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.internal.util.logging.formatter;
+
+import java.util.Collection;
+
+/**
+ * Used with JBoss Logging to display collection of class names in log messages.
+ *
+ * @author Guillaume Smet
+ */
+public class CollectionOfClassesObjectFormatter {
+
+	private final String stringRepresentation;
+
+	public CollectionOfClassesObjectFormatter(Collection<Class<?>> classes) {
```

Comment:
Or  this:

    stringRepresentation = classes.stream()
        .map( c -> c.getName() )
        .collect( Collectors.joining( ", " ) );

Sorry, couldn't resist :)